webcomponents/index.ts

```diff
@@ -0,0 +1,16 @@
+import JoyRtcComponent from './model/joy-rtc'
+import { DOMAttributes } from "react"
```

Comment:
This web-components package don't need react

Maybe We're need a better `.d.ts` file for typescript